stdlib/2and3/decimal.pyi

```diff
@@ -209,94 +213,97 @@ class Context:
         clamp: int
     else:
         _clamp: int
-    traps: Dict[type, bool]
-    flags: Any
+    traps: Dict[_TrapType, bool]
+    flags: Dict[_TrapType, bool]
     if sys.version_info >= (3,):
         def __init__(self, prec: Optional[int] = ..., rounding: Optional[str] = ...,
                      Emin: Optional[int] = ..., Emax: Optional[int] = ...,
                      capitals: Optional[int] = ..., clamp: Optional[int] = ...,
-                     flags=..., traps=...,
-                     _ignored_flags=...) -> None: ...
+                     flags: Union[None, Dict[_TrapType, bool], Container[_TrapType]] = ...,
+                     traps: Union[None, Dict[_TrapType, bool], Container[_TrapType]] = ...,
+                     _ignored_flags: Optional[List[_TrapType]] = ...) -> None: ...
     else:
         def __init__(self, prec: Optional[int] = ..., rounding: Optional[str] = ...,
-                     traps=..., flags=...,
+                     traps: Union[None, Dict[_TrapType, bool], Container[_TrapType]] = ...,
+                     flags: Union[None, Dict[_TrapType, bool], Container[_TrapType]] = ...,
                      Emin: Optional[int] = ..., Emax: Optional[int] = ...,
                      capitals: Optional[int] = ..., _clamp: Optional[int] = ...,
-                     _ignored_flags=...) -> None: ...
+                     _ignored_flags: Optional[List[_TrapType]] = ...) -> None: ...
     if sys.version_info >= (3,):
-        def __delattr__(self, name): ...
-        def __reduce__(self): ...
-    def clear_flags(self): ...
+        def __setattr__(self, name: str, value: Any) -> Any: ...
```

Comment:
I think this should actually be omitted because the effect of having it is that the type checker will think setting any attribute is allowed. However, the implementation actually only allows the attributes that are known to exist on the class.

Reply:
I replaced it with a comment saying so, for future reference.